### DISCUSSION TITLE
Is it possible to force grabbing of artist/album images without clicking on them?

### DISCUSSION THREAD

**Discussion by fernplant** [music-assistant/support#988] [Q&A] [answered]
Just updated to latest and lost all the images. Im a bit anal retentive and like to see all the images for artists :) Anyway to force them to get pulled without having to click on each one?

**Comment by marcelveldt** [comment, marked as answer]
Just wait, they will auto load in time. Next update (December) will improve this

**Reply by fernplant** [reply]
Great, thank you!

**Reply by fernplant** [reply]
Hm. It actually did not update. It also has not updated like that in past versions either. It will only grab the artist image once I click on the artist

**Reply by marcelveldt** [reply]
It will update once you open the artist for the first time or play any of its music. It will not auto grab metadata by itself. I will provide that feature in the feature but still with some throttling as we do not want to stress the metadata providers.

**Reply by fernplant** [reply]
Sounds reasonable. Thanks

**Comment by OzGav** [comment]
Relates to v1
